release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+to be part of Kubernetes and evolve in lock-step with the platform.
+
+CRI-O brings:
+
+* A minimal and secure architecture.
+* Excellent scale and performance.
+* The ability to run any Open Container Initiative (OCI) or docker image.
+* Familiar operational tooling and commands.
+
+image::crio-3-7.png[CRI-O]
+
+[[ocp-37-cluster-wide-tolerations-per-namespace-tolerations]]
+==== Cluster-wide Tolerations and Per-namespace Tolerations to Control Pod Placement
+
+In a multi-tenant environment, you want to leverage administration controllers
+to help define rules that can help govern a cluster, should a tenant not set a
+toleration for placement.
+
+The following is offered to administrators where the namespace setting will
+override the cluster setting:
+
+* Cluster-wide and per-namespace default toleration for pods.
+* Cluster-wide and per-namespace white-listing of toleration for pods.
+
+.Cluster-wide Off Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        kind: DefaultAdmissionConfig
+        apiVersion: v1
+        disable: true
+----
+
+.Cluster-wide On Example
+----
+admissionConfig:
+  pluginConfig:
+    PodTolerationRestriction:
+      configuration:
+        apiVersion: podtolerationrestriction.admission.k8s.io/v1alpha1
+        kind: Configuration
+        default:
+         - key: key3
+           value: value3
+        whitelist:
+         - key: key1
+           value: value1
+         - key: key3
+           value: value3
+----
+
+.Namespece-specific Example
+----
+piVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: ""
+    openshift.io/display-name: ""
+    openshift.io/sa.scc.mcs: s0:c8,c7
+    openshift.io/sa.scc.supplemental-groups: 1000070000/10000
+    openshift.io/sa.scc.uid-range: 1000070000/10000
+    scheduler.alpha.kubernetes.io/defaultTolerations: '[ { "key": "key1", "value":"value1" }]'
+    scheduler.alpha.kubernetes.io/tolerationsWhitelist: '[ { "key": "key1", "value":
+      "value1" }, { "key": "key2", "value": "value2" } ]'
+  generateName: dma-
+spec:
+  finalizers:
+  - openshift.io/origin
+  - kubernetes
+----
+
+[[ocp-37-security]]
+=== Security
+
+[[ocp-37-documented-private-public-key-configurations-and-crypto-levels]]
+==== Documented Private and Public Key Configurations and Crypto Levels
+
+{product-title} is a secured by default implementation of Kubernetes.
+
+{product-title} leverages Transport Layer Security (TLS) cipher suites, JSON Web
+Algorithms (JWA) crypto algorithms, and offers external libraries such as The
+Generic Security Service Application Program Interface (GSSAPI) and libgpgme.
+
+xref:../architecture/index.adoc#architecture-index[Private and public key
+configurations and Crypto levels] are now documented for {product-title}.
+
+[[ocp-37-node-authorizer-node-restriction-admission-plug-in]]
+==== Node Authorizer and Node Restriction Admission Plug-in
```

Comment:
We should link to the upgrade docs that cover the post upgrade step documenting the commands listed by the release notes (making sure its shown in 2 places helps us work / deal with cases when they come in for stuff like this).

Reply:
@sferich888 I see the command documented in the Admin Guide --> Managing Projects section. Is that the section you mean? If you can send me a link to what you have in mind, that would be great.